doc/reference/reference_lua/msgpack.rst

```diff
@@ -241,7 +236,7 @@ Below is a list of all ``msgpack`` functions and members.
 
 .. function:: decode_array_header(byte-array, size)
 
-    Call the mp_decode_array function in the `MsgPuck <http://rtsisyk.github.io/msgpuck/>`_ library
+    Call the `MsgPuck <https://rtsisyk.github.io/msgpuck/>`_'s ``mp_decode_array`` function
```

Comment:
May be worth mentioning here that this function expects the data to be a MsgPack array. If it isn't, an exception is raised. Same comment for `decode_map_header`.